src/cueUtilities.js

```diff
@@ -319,60 +320,83 @@ module.exports = function (params, cy, api) {
           }
         });
 
-        cy.on('tap', 'node', data.eTap = function (event) {
-          var node = this;
-          var opts = options();
-          let nodeHasCue = isSelectedGroupsContains(node);
-          if (hoveredGroup) {
-            nodeHasCue = nodeHasCue || hoveredGroup.id() === node.id();
+        function clickedOnCueRegion(node, event) {
```

Comment:
Since this is being used for more than just clicks, maybe call it `eventOverCueRegion` or something similar. Change calling functions similarly.